crates/bevy_ui/src/flex/mod.rs

```diff
@@ -179,7 +179,11 @@ pub fn flex_node_system(
 
     // assume one window for time being...
     let logical_to_physical_factor = if let Some(primary_window) = windows.get_primary() {
-        primary_window.scale_factor()
```

Comment:
Or just make `Window::scale_factor()` return `1` on a physical sized window?

Reply:
I kept `Window::scale_factor()` as is so people who opt out of hidpi can still access the scale factor when implementing an alternative system.

Reply:
That makes sense. I just noticed that you have already reimplemented that same logic three times, which isn't ideal. Maybe `monitor_scale_factor` vs `[window_]scale_factor` would be sensible? It's unclear what would be best.

Reply:
Diffrent windows can have diffrent scale factors, so I would choose window_scaling_factor.